app/components/Map.js

```diff
@@ -114,6 +109,17 @@ export default class Map extends Component {
       );
   }
 
+  moveMapToUser() {
+    this.getCurrentLocation( (coords) => {
+      this.refs.map.animateToRegion(coords, 300);
+    });
+  }
```

Comment:
This eats a lot of CPU in my simulator and feels slow. :frowning: Can we reduce the time on this and goToTarget to 100ms?